build.sh

```diff
@@ -0,0 +1,199 @@
+#!/bin/bash
+
```

Comment:
./build.sh pylibwholegraph
Building for *ALL* supported GPU architectures...
**env: ‘setup.py’: No such file or directory**